packages/cli/src/commands/migrate/tasks/convert.ts

```diff
@@ -46,14 +46,11 @@ export function convertTask(options: MigrateCommandOptions, logger: Logger): Lis
         const reportOutputPath = resolve(options.basePath, options.outputPath);
         generateReports('migrate', reporter, reportOutputPath, options.format);
 
-        const { totalErrorCount, errorFixedCount, hintAddedCount } = getReportSummary(
-          reporter.report
-        );
+        const { totalErrorCount, hintAddedCount } = getReportSummary(reporter.report);
         const migratedFileCount = migratedFiles.length;
         task.title = `${migratedFileCount} JS ${
           migratedFileCount === 1 ? 'file' : 'files'
-        } has been converted to TS. There are ${totalErrorCount} errors caught by rehearsal:
-                - ${errorFixedCount} have been fixed automatically by rehearsal.
+        } has been converted to TS. There are ${totalErrorCount} errors caught by rehearsal
                 - ${hintAddedCount} have been updated with @ts-expect-error @rehearsal TODO which need further manual check.`;
```

Comment:
```suggestion
        } has been converted to TS. There were ${totalErrorCount} errors caught by rehearsal:
                - ${totalErrorCount - hintAddedCount} have been fixed automatically by rehearsal.
                - ${hintAddedCount} have been updated with @ts-expect-error which need to be manually migrated.`;
```

Reply:
@chadhietala In this PR, totalErrorCount is the total error count after we auto fixed the original errors. totalErrorCount here is in most cases identical to hintAddedCount. We are currently not reporting anything from auto fix. I can add code in another PR in diagnostic-fix plugin to report the number of original errors before auto fix. Please let me know if it's OK.